indico/htdocs/sass/base/_theme.scss

```diff
@@ -145,3 +145,21 @@
         color: $gray;
     }
 }
+
+.semantic-background {
+    &.submitted {
+        background-color: $blue;
+    }
+
+    &.accepted {
+        background-color: $green;
+    }
+
+    &.rejected {
+        background-color: $red;
+    }
+
+    &.to-be-corrected {
+        background-color: $yellow;
+    }
+}
```

Comment:
This is not what I meant last time! I meant to create a class that extends from `%semantic-background` like this:
```scss
.semantic-background {
    @extend %semantic-background;
}
```
So that when you want to have something with a blue background, you can do:
```html
<div class="semantic-background highlight">...</div>
```
The classes `submitted`, `accepted`, etc do not belong in the `_theme.scss` file. The solution to your problem is to go to the file `indico/modules/events/papers/models/revisions.py` and replace the `__css_classes__` row with this:
```python
__css_classes__ = [None, 'highlight', 'success', 'error', 'warning']
```